docs/src/components/base/borders/borders--2px.html

```diff
@@ -0,0 +1,7 @@
+{% for color in colors %}
```

Comment:
Can we try and keep the naming patterns for files consistent? This can be the same as our variable naming scheme described in the [front end guide](https://pages.18f.gov/frontend/#suggested-custom-methodology) and our contributing doc.

So they should all be `borders-4px.html`, there shouldn't be any BEM style `--`s.